packages/typespec-test/test/healthInsights_radiologyinsights/generated/typespec-ts/README.md

```diff
@@ -1,46 +1,74 @@
+[Package (NPM)](https://www.npmjs.com/package/@azure-rest/health-insights-radiologyinsights) |
+[API reference documentation](https://docs.microsoft.com/javascript/api/@azure-rest/health-insights-radiologyinsights?view=azure-node-preview) |
 
 ## Getting started
 
 ### Currently supported environments
 
-- LTS versions of Node.js
+- [LTS versions of Node.js](https://github.com/nodejs/release#release-schedule)
+- Latest versions of Safari, Chrome, Edge and Firefox.
+
+See our [support policy](https://github.com/Azure/azure-sdk-for-js/blob/main/SUPPORT.md) for more details.
 
 ### Prerequisites
 
-- You must have an [Azure subscription](https://azure.microsoft.com/free/) to use this package.
+- An [Azure subscription][azure_sub].
 
 ### Install the `@azure-rest/health-insights-radiologyinsights` package
 
-Install the AzureHealthInsights REST client REST client library for JavaScript with `npm`:
+Install the AzureHealthInsights client library for JavaScript with `npm`:
 
 ```bash
 npm install @azure-rest/health-insights-radiologyinsights
 ```
 
 ### Create and authenticate a `AzureHealthInsightsClient`
 
-To use an [Azure Active Directory (AAD) token credential](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/identity/identity/samples/AzureIdentityExamples.md#authenticating-with-a-pre-fetched-access-token),
-provide an instance of the desired credential type obtained from the
-[@azure/identity](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/identity/identity#credentials) library.
+To create a client object to access the AzureHealthInsights API, you will need the `endpoint` of your AzureHealthInsights resource and a `credential`. The AzureHealthInsights client can use Azure Active Directory credentials to authenticate.
+You can find the endpoint for your AzureHealthInsights resource in the [Azure Portal][azure_portal].
+
+You can authenticate with Azure Active Directory using a credential from the [@azure/identity][azure_identity] library or [an existing AAD Token](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/identity/identity/samples/AzureIdentityExamples.md#authenticating-with-a-pre-fetched-access-token).
+
+To use the [DefaultAzureCredential][defaultazurecredential] provider shown below, or other credential providers provided with the Azure SDK, please install the `@azure/identity` package:
 
-To authenticate with AAD, you must first `npm` install [`@azure/identity`](https://www.npmjs.com/package/@azure/identity) 
+```bash
+npm install @azure/identity
+```
+
+You will also need to **register a new AAD application and grant access to AzureHealthInsights** by assigning the suitable role to your service principal (note: roles such as `"Owner"` will not grant the necessary permissions).
+Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables: `AZURE_CLIENT_ID`, `AZURE_TENANT_ID`, `AZURE_CLIENT_SECRET`.
```

Comment:
we shall not suggesting user to use 'client_secret' moving forward.

Reply:
created issue for it https://github.com/Azure/autorest.typescript/issues/2882